services/name-gender-bio.js

```diff
@@ -1,5 +1,4 @@
-const mongoose = require("mongoose");
-let Profile = mongoose.model("profiles");
+const Profile = require("../models/Profile");
 
 module.exports.addGender = (id, gender) => {
   return new Promise((resolve, reject) => {
```

Comment:
this also can be converted into async/await, and there is a PR for this function that was not merged yet, that removes the gender constraint -  and I did not convert it into async/await lol -  if you want you can rewrite this and just remove the gender check. both M and F should be saved. if not, it's ok, I can do it in my PR that is hanging there